VerilogCodingStyle.md

```diff
@@ -1527,6 +1526,33 @@ Do not use positional arguments to connect signals to ports.
 
 Instantiate ports in the same order as they are defined in the module.
 
+Align expressions in [tabular style](#tabular-alignment).
+
+***Use named parameters for all instantiations.***
+
+When parameterizing an instance, specify the parameter using the named parameter
+style. An exception is if there is only one parameter that is obvious such as
+register width, then the instantiation can be implicit.
+
+Indentation for module instantiation follows the standard indentation
+rule of two space indentation.
+
+```systemverilog
+my_module #(
+  .Height(5),
+  .Width(10)
```

Comment:
We should also tabular-align parameter set.

Reply:
Yes, but let's handle that in a separate PR. I'll push that as soon as we have merged this one to avoid the merge conflict.